images/lib/refinery/images/configuration.rb

```diff
@@ -25,51 +18,18 @@ module Images
       :medium => '225x255>',
       :large => '450x450>'
     }
-
     self.whitelisted_mime_types = %w[image/jpeg image/png image/gif image/tiff]
-
     self.image_views = [:grid, :list]
     self.preferred_image_view = :grid
 
-    # We have to configure these settings after Rails is available.
-    # But a non-nil custom option can still be provided
-    class << self
-      def datastore_root_path
-        config.datastore_root_path || (Rails.root.join('public', 'system', 'refinery', 'images').to_s if Rails.root)
-      end
-
-      def s3_backend
-        config.s3_backend.presence || Core.s3_backend
-      end
-
-      def s3_bucket_name
-        config.s3_bucket_name.presence || Core.s3_bucket_name
-      end
-
-      def s3_access_key_id
-        config.s3_access_key_id.presence || Core.s3_access_key_id
-      end
-
-      def s3_secret_access_key
-        config.s3_secret_access_key.presence || Core.s3_secret_access_key
-      end
-
-      def s3_region
-        config.s3_region.presence || Core.s3_region
-      end
-
-      def custom_backend?
-        config.custom_backend_class.nil? ? Core.dragonfly_custom_backend? : config.custom_backend_class.present?
-      end
+    # Images should always use these changes to the dragonfly defaults
 
-      def custom_backend_class
-        config.custom_backend_class.nil? ? Core.dragonfly_custom_backend_class : config.custom_backend_class.constantize
-      end
+    # Dragonfly processor to strip image of all profiles and comments (imagemagick converts -strip)
+    strip = -> content {content.process!(:convert, '-strip')}
 
-      def custom_backend_opts
-        config.custom_backend_opts.presence || Core.dragonfly_custom_backend_opts
-      end
+    self.dragonfly_name         = :refinery_images
+    self.dragonfly_processors   = [{name: :strip, block: strip}]
```

Comment:
Primarily to avoid adding the function inside the module, could `strip` be inlined here?

```ruby
self.dragonfly_processors = [{
  name: :strip,
  block: -> (content) { content.process!(:convert, '-strip') }
}]
```